lm_eval/tasks/mlsum_de.py

```diff
@@ -0,0 +1,115 @@
+import datasets
+from lm_eval.base import rf, Task
+from functools import partial
+
+
+def _mlsum_metric(predictions, references):
+    summarization_metric = datasets.load_metric("rouge")
+    return summarization_metric.compute(predictions=predictions, references=references, use_agregator=True)
+
+
+def _mlsum_agg(key, items):
+    predictions, references = zip(*items)
+    return _mlsum_metric(predictions=predictions, references=references)[key].mid.fmeasure
+
+
+class MLSUM(Task):
+    VERSION = None
+    DATASET_PATH = "mlsum"
+    DATASET_NAME = 'de'
```

Comment:
What about adding the other languages as well? See for example https://github.com/OpenGPTX/lm-evaluation-harness/blob/master/lm_eval/tasks/xquad.py